substrate/frame/support/procedural/src/construct_runtime/expand/metadata.rs

```diff
@@ -117,7 +117,7 @@ pub fn expand_runtime_metadata(
 					pallets: #scrate::__private::vec![ #(#pallets),* ],
 					extrinsic: #scrate::__private::metadata_ir::ExtrinsicMetadataIR {
 						ty,
-						version: <#extrinsic as #scrate::sp_runtime::traits::ExtrinsicMetadata>::VERSION,
+						versions: #scrate::__private::vec![ <#extrinsic as #scrate::sp_runtime::traits::ExtrinsicMetadata>::VERSION ],
```

Comment:
Maybe similar to what Niklas asked but, I guess this should now contain 2 versions, 4 and 5, as of https://github.com/paritytech/polkadot-sdk/pull/3685 merging?

Reply:
`ExtrinsicMetadata::VERSION` should be a vec itself (or static slice).